cms/templates/js/add-list-item.underscore

```diff
@@ -0,0 +1,3 @@
+<button class="action action-add">
```

Comment:
Yes, it's only used in one place. I suppose it doesn't need its own template... are you suggesting to inline it into `list.js`?